addons/website_event_track/controllers/event_track.py

```diff
@@ -80,10 +80,13 @@ def event_tracks(self, event, tag=None, **searches):
             # TODO: remove in a few stable versions (v19?), including the "prevent_redirect" param in templates
             slug = request.env['ir.http']._slug
             return request.redirect(f'/event/{slug(event)}/track', code=301)
+        view = [menu for menu in event.track_menu_ids if menu['display_name'] == 'Talks'][0].view_id
```

Comment:
Here, both `talks` and `agenda` menu have their menu_type equal to `track`. Hence, using `menu_id.name`.

Reply:
@nihp-odoo oh I see indeed "track" adds those menus :thinking: 

Maybe we should use the website menu url then (`track_menu_ids.filtred(lambda menu: menu.menu_id.url.endswith('/agenda'))`)? As the names are translated terms I don't think it will be reliable otherwise.

Or as it's master maybe we can introduce a subtype on these menus to differentiate them without checking the url...